src/main/scala/mesosphere/marathon/state/Group.scala

```diff
@@ -264,7 +264,7 @@ object Group extends StrictLogging {
 
       // Only top-level groups are allowed to set the enforce role parameter.
       if (group.enforceRole.contains(true)) {
-        group.id.map(_.toPath) is optional(PathId.topLevel)
+        group.id.map(base.append(_)) is optional(PathId.topLevel)
```

Comment:
I actually think the group.id needs to be treated as the base path if it's empty.